docs/development/reviewing-patches.rst

```diff
@@ -34,6 +34,47 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+Testing branches remotely
+-------------------------
+
+It can be helpful to check out a branch with a PR on it, in order to test
+how the changes in it work on your machine. To do so, first pull
+down the PR branch by running the following commands:
```

Comment:
The first time you use the abbrevation "PR" please spell it out as "pull request".